Miracast/MiracastService/MiracastController.cpp

```diff
@@ -0,0 +1,1968 @@
+                            m_groupInfo = new GroupInfo;
+                            size_t found = event_buffer.find("client");
+                            size_t found_space = event_buffer.find(" ");
+
+                            MIRACASTLOG_TRACE("CONTROLLER_GO_GROUP_STARTED Received\n");
+
+                            if (found != std::string::npos)
+                            {
+                                MIRACASTLOG_INFO("!!!! P2P GROUP STARTED IN CLIENT MODE !!!!");
+                                m_groupInfo->ipAddr = parse_p2p_event_data(event_buffer.c_str(), "ip_addr");
+                                m_groupInfo->ipMask = parse_p2p_event_data(event_buffer.c_str(), "ip_mask");
+                                m_groupInfo->goIPAddr = parse_p2p_event_data(event_buffer.c_str(), "go_ip_addr");
+                                m_groupInfo->goDevAddr = parse_p2p_event_data(event_buffer.c_str(), "go_dev_addr");
+                                m_groupInfo->SSID = parse_p2p_event_data(event_buffer.c_str(), "ssid");
+
+                                size_t found_client = event_buffer.find("client");
+                                m_groupInfo->interface = event_buffer.substr(found_space, found_client - found_space);
+                                REMOVE_SPACES(m_groupInfo->interface);
+
+                                if (getenv("GET_PACKET_DUMP") != nullptr)
+                                {
+                                    std::string tcpdump;
+                                    tcpdump.append("tcpdump -i ");
+                                    tcpdump.append(m_groupInfo->interface);
+                                    tcpdump.append(" -s 65535 -w /opt/p2p_cli_dump.pcap &");
+                                    MIRACASTLOG_VERBOSE("Dump command to execute - %s", tcpdump.c_str());
+                                    system(tcpdump.c_str());
+                                }
+
+                                std::string default_gw_ip = "";
+
+                                // STB is a client in the p2p group
+                                m_groupInfo->isGO = false;
+                                m_groupInfo->localIPAddr = start_DHCPClient(m_groupInfo->interface, default_gw_ip);
+                                if (m_groupInfo->localIPAddr.empty())
+                                {
+                                    MIRACASTLOG_ERROR("Local IP address is not obtained");
+                                }
+                                else
+                                {
+                                    if (m_groupInfo->goIPAddr.empty())
+                                    {
+                                        MIRACASTLOG_VERBOSE("default_gw_ip [%s]\n", default_gw_ip.c_str());
+                                        m_groupInfo->goIPAddr.append(default_gw_ip);
+                                    }
+                                    #ifdef NATIVE_PLAYER_INTEGRATED
```

Comment:
when this flag will be enabled or disabled

Reply:
It flag will not be enabled. I removed it as part of https://github.com/rdkcentral/rdkservices/commit/019f3d11389a6271609f0a3153b3a3984f89ca64